UI/src/less/core/forms.less

```diff
@@ -634,6 +634,10 @@ input[type="checkbox"] {
         display: none;
       }
     }
+    .multiSelectItems{
+      text-align: center;
+      margin-left: -10px
```

Comment:
Maybe doesn't cause anything but add the ';' in this

margin-left: -10px;

Reply:
Done